t/JustPod_corpus.t

```diff
@@ -58,99 +58,129 @@ BEGIN {
   plan tests => scalar @test_files;
 }
 
+@test_files = sort @test_files;
+
+my @skip_on_windows = qw{
+  t/corpus/8859_7.pod
+  t/corpus/laozi38p.pod
+  t/junk2.pod
+  t/perlcyg.pod
+  t/perlfaq.pod
+  t/perlvar.pod
+  t/search60/A/x.pod
+  t/search60/B/X.pod
+  t/testlib1/hinkhonk/Glunk.pod
+  t/testlib1/pod/perlflif.pod
+  t/testlib1/pod/perlthng.pod
+  t/testlib1/squaa/Glunk.pod
+  t/testlib1/zikzik.pod
+  t/testlib2/hinkhonk/Glunk.pod
+  t/testlib2/pod/perlthng.pod
+  t/testlib2/pod/perlzuk.pod
+  t/testlib2/pods/perlzoned.pod
+  t/testlib2/squaa/Wowo.pod
+};
+
+my $is_windows = $^O eq 'MSWin32';
 foreach my $file (@test_files) {
-  my $parser = Pod::Simple::JustPod->new();
-  $parser->complain_stderr(0);
-
-  my $input;
-  open( IN , '<:raw' , $file ) or die "$file: $!";
-  $input .= $_ while (<IN>);
-  close( IN );
-
-  my $output;
-  $parser->output_string( \$output );
-  $parser->parse_string_document( $input );
-
-  if ($parser->any_errata_seen()) {
-    pass("Skip '$file' because of pod errors");
-    next if "$]" lt '5.010.001';     # note() not found in earlier versions
-    my $errata = $parser->errata_seen();
-    foreach my $line_number (sort { $a <=> $b } keys %$errata) {
-        foreach my $err_msg (sort @{$errata->{$line_number}}) {
-            note("$file: $line_number: $err_msg");
-        }
+  SKIP: {
```

Comment:
Doing `skip(..) and next if $conditions` would mean the SKIP block isn't needed, which removes the extra indentation and the whitespace changes on every line.

Reply:
not sure what's your complain here, there is `Hide whitespace change` at the top
these tests were broken under windows and need investigation, but this was outside the scope of this change

Reply:
This is not a complaint.

I'm observing that a number of lines are changing only in whitespace, which are nice to avoid when possible. This affects functionality like 'git blame'.

Reply:
ok I see thanks for this precision.